spec/support/vcr.rb

```diff
@@ -1,21 +1,21 @@
 VCR.configure do |c|
```

Comment:
I have opened an issue, let's continue the discussion here:
https://github.com/glossier/solidus_retail/issues/27